internal/obsreportconfig/obsmetrics/obs_exporter.go

```diff
@@ -7,6 +7,9 @@ const (
 	// ExporterKey used to identify exporters in metrics and traces.
 	ExporterKey = "exporter"
 
+	// DataTypeKey used to identify the data type in the queue size metric.
+	DataTypeKey = "data_type"
```

Comment:
should this be what the [otep](https://github.com/open-telemetry/oteps/pull/259/files#diff-8ab369bae50cfda348c0bd20882db8a1bdbbffb979abc70bff298192a948588bR218) recommends `otel.signal`?

Reply:
I'm not sure about it. We use `data_type` as a field for logging. Once the OTEP is ready and we adopt it, the metric is going to change anyway. Keeping an attribute name doesn't help with the migration. Also, `otel.signal` could still change before the OTEP is merged?

Reply:
@codeboten I'm open to any attribute name. Let me know what do you think considering my previous comment.

Reply:
@dmitryax probably ok to go with an attribute consistent w/ logging for now.